packages/badge/components/badge.tsx

```diff
@@ -11,23 +11,17 @@ export type BadgeAppearance =
 
 export interface BadgeProps {
   appearance?: BadgeAppearance;
-  children: React.ReactNode;
-}
-
-export class Badge extends React.PureComponent<BadgeProps, {}> {
-  public static defaultProps: Partial<BadgeProps> = {
-    appearance: "default"
-  };
-
-  public render() {
-    const { appearance, children } = this.props;
-
-    return (
-      <span className={badge(appearance)} data-cy="badge">
-        {children}
-      </span>
-    );
-  }
+  children: React.ReactNode | string;
 }
 
+const Badge: React.FC<BadgeProps> = ({
+  appearance = "default",
+  children
+}: BadgeProps) => {
+  return (
+    <span className={badge(appearance)} data-cy="badge">
+      {children}
+    </span>
+  );
+};
```

Comment:
so, as I understand it, to strictly recreate a Pure Component, we'll need to memoize it by wrapping it in React memo: https://stackoverflow.com/questions/43470659/declare-a-functional-component-as-pure